arithmetization/src/main/java/net/consensys/linea/zktracer/module/rlptxn/RlpTxn.java

```diff
@@ -593,7 +593,7 @@ private void handlePhaseBeta(RlpTxnColumnsValue traceValue, Transaction tx, Trac
     checkArgument(bigIntegerToBytes(V).size() <= 8, "V is longer than 8 bytes");
     final boolean betaIsZero =
         V.equals(BigInteger.valueOf(27))
-            || V.equals(BigInteger.valueOf(28)); // beta = ChainId = 0 iff (V == 27 or V == 28)
+            || V.equals(BigInteger.valueOf(28)); // beta = 0 iff (V == 27 or V == 28)
```

Comment:
My understanding is that if v $\in$ { 27, 28 } then _we cannot draw any conclusions as to the value of_ $\beta$

Reply:
These transactions don't have replay protection and so can be replayed on any chain (that allows for such transactions.)

Reply:
we have a beta, but then beta is NOT the value of the chainId ...